tests/11-file-deletion-tests/testcafe/file_deletion_tests.spec.js

```diff
@@ -56,10 +56,11 @@ test('Migrate Files to be Deleted', async t => {
     let fileListing = 'https://islandora-idc.traefik.me/admin/content/files/';
     let mediaListing = 'https://islandora-idc.traefik.me/admin/content/media';
 
-
     // Navigate to Files listing, verify File is present, and used in one place
     await t.navigateTo(fileListing);
-    const fileToDelete = Selector('div.view-content').find('a').withText('Test Geo Tif File');
```

Comment:
Cool, these changes are presumably all related to the file name field changing.  So now when a file is deleted, it's gone.